src/app/(BottomTabNavigation)/Profile/LogOut/styles.ts

```diff
@@ -0,0 +1,79 @@
+import { StyleSheet } from 'react-native';
+
+import { colors } from '../../../../styles/colors';
+
+export default StyleSheet.create({
+  container: {
```

Comment:
In terms of readability, it'd be helpful to organize our style sheets so larger containers are at the top, then component styles follow, and finally text styling at the bottom. This is sort of opinionated but it makes it easier to review if everything is consistent.

Reply:
changed!